ir/src/symbol_table.rs

```diff
@@ -151,6 +159,25 @@ impl SymbolTable {
         Ok(())
     }
 
+    /// Adds all random values by their identifier names and array length.
+    pub(super) fn insert_random_values(
+        &mut self,
+        values: &RandomValues,
+    ) -> Result<(), SemanticError> {
+        // TODO: add name of random values vector to the identifiers. For now it can be called only
+        // by `$rand`
```

Comment:
Is this left out of this PR because it's quite complicated to do? If we're not including it here, can you create a sub-issue for it so we don't lose track of it?

Reply:
Initially, I decided to leave the implementation of this for later, because the PR turned out to be quite large, and I did not want to complicate it with additional optional functionality. But we split it into two, so I don't see any more problems not including it in the current PR. So I'll do it now!